docs/references/room.md

```diff
@@ -153,58 +147,55 @@ await room.say(miniProgram);
 
 #### Example _\(Event:join \)_
 
-```javascript
-const bot = new Wechaty()
+```python
+bot = Wechaty()
 await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'topic of your room'}) // change `event-room` to any room topic in your wechat
-if (room) {
-  room.on('join', (room, inviteeList, inviter) => {
-    const nameList = inviteeList.map(c => c.name()).join(',')
-    console.log(`Room got new member ${nameList}, invited by ${inviter}`)
-  })
-}
+# after logged in...
+room = await bot.Room.find("wechaty") # change `event-room` to any room topic in your wechat
+def on_join(invitee_list, inviter):
+  log.info('room.on(join) id:', room.room_id)
+  check_room_join(bot, room, invitee_list, inviter)
+if room:
+  room.on('join', on_join)
 ```
 
 #### Example _\(Event:leave \)_
 
-```javascript
-const bot = new Wechaty()
+```python
+bot = Wechaty()
 await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'topic of your room'}) // change `event-room` to any room topic in your wechat
-if (room) {
-  room.on('leave', (room, leaverList) => {
-    const nameList = leaverList.map(c => c.name()).join(',')
-    console.log(`Room lost member ${nameList}`)
-  })
-}
+# after logged in...
+room = await bot.Room.find("wechaty") # change `event-room` to any room topic in your wechat
+def on_leave(leaver_list, remover):
+  log.info('Bot' + 'Room EVENT: leave - "%s" leave(remover "%s"), bye bye' % (','.join(leaver_list), remover or 'unknown'))
+if room:
+  room.on('leave', on_leave)
 ```
 
 #### Example _\(Event:topic \)_
 
-```javascript
-const bot = new Wechaty()
+```python
+bot = Wechaty()
 await bot.start()
-// after logged in...
-const room = await bot.Room.find({topic: 'topic of your room'}) // change `event-room` to any room topic in your wechat
-if (room) {
-  room.on('topic', (room, topic, oldTopic, changer) => {
-    console.log(`Room topic changed from ${oldTopic} to ${topic} by ${changer.name()}`)
-  })
-}
+# after logged in...
+room = await bot.Room.find("wechaty") # change `event-room` to any room topic in your wechat
+def on_topic(topic, old_topic, changer):
```

Comment:
`on_topic` method should be async-based, otherwise it will occur error at running time.